crates/outlayer/proto/proto/outlayer.proto

```
@@ -0,0 +1,68 @@
+syntax = "proto3";
+package outlayer;
```

Buf (1.70.0)

[error] Files with package "outlayer" must be within a directory "outlayer" relative to root but were in directory "crates/outlayer/proto/proto".

(PACKAGE_DIRECTORY_MATCH)